extensions/adobe/experience/decisioning/interaction-measurement-details.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/interaction-measurement",
```

Comment:
New datatype tha defines how measurements for decision outcomes are performed

Reply:
@jbeckert is there a WIKI describing this particular data type and why we might need it?